src/polyfill.ts

```diff
@@ -183,9 +183,16 @@ export async function polyfill() {
 
   if (Object.values(rules).length) {
     // apply polyfill
-    // @@@ Wait for DOM load?
-    // Or make `autoUpdate` work.
-    position(rules);
+    if (
+      document.readyState === 'loading' ||
+      document.readyState === 'interactive'
+    ) {
+      document.addEventListener('load', () => {
```

Comment:
Does this work? I thought it only fires on the `window`:

```suggestion
      window.addEventListener('load', () => {
```

Reply:
You can test this by adding a new `<script>` tag near the beginning of the `<head>` in `index.html`:

```
    <script>
      console.log(document.readyState);
      document.addEventListener('load', () => {
        console.log('loaded');
      });
    </script>
```

Reply:
oh whoops, good catch